### PR TITLE
[stable/rabbitmq-ha] Fix namespace and labels in prometheus alerts definition

### PR DESCRIPTION
**Pull request by jkrcma**
**What this PR does / why we need it**:
Fixes the installation of Prometheus Operator chart manifests in a different than default namespace

### CODE REVIEW

**Comment by cpanato** [comment]
@jkrcma please sign the DCO - https://www.helm.sh/blog/helm-dco/index.html

**Comment by jkrcma** [comment]
@cpanato Accidentally used the wrong email without the GPG, fixed

**Comment by jkrcma** [comment]
@cpanato I found more bug in the alerting template, is it okay to attach the fix in another commit in this PR?

**Comment by cpanato** [comment]
sure @jkrcma but if you adding more stuffs please edit the title and other things as well 😄

**Comment by steven-sheehy** [comment]
Duplicate of #6967, but that one is not merge-able and author non-responsive.

**Comment by jkrcma** [comment]
Amended the commit to fix the value of `app` label

@steven-sheehy I am ready to make this PR merged quickly :)

File: stable/rabbitmq-ha/templates/alerts.yaml [hunk 1]
```diff
@@ -3,12 +3,12 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-rabbitmq-alerts
-  namespace: monitoring
+  namespace: {{ .Values.prometheus.operator.serviceMonitor.namespace }}
   labels:
-    app: "rabbitmq"
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    heritage: {{ .Release.Service }}
```
Comment (davidkarlsen) [inline stable/rabbitmq-ha/templates/alerts.yaml:10]: Won’t label changes break upgrades?

Comment (jkrcma) [reply stable/rabbitmq-ha/templates/alerts.yaml:10]: @davidkarlsen Possibly, I was trying to `helm upgrade` the running installation while enabling Prometheus alerts which should be only matter of adding this ConfigMap, but had to eventually stick to `--force` option. The point is with this wrong `app` label the chart doesn't do what it is supposed to do

**Comment by jkrcma** [comment]
@mattfarina @paulczar Bump please

File: stable/rabbitmq-ha/templates/alerts.yaml (hunk 1 above)
Comment (steven-sheehy) [reply stable/rabbitmq-ha/templates/alerts.yaml:10]: Maybe bump major or minor version to indicate it's a breaking change?

Comment (jkrcma) [reply stable/rabbitmq-ha/templates/alerts.yaml:10]: Makes sense, changed

Comment (mattfarina) [reply stable/rabbitmq-ha/templates/alerts.yaml:10]: As long at the change isn't in the matchLabels or other place where the labels are immutable it should be ok.

**Comment by mattfarina** [comment]
@jkrcma This looks good to me but it needs to be fixed per master. Can you resolve the conflict and then I'm happy to approve.

cc @etiennetremel

**Comment by jkrcma** [comment]
@mattfarina Rebased, thanks!

**Comment by jkrcma** [comment]
/assign @mattfarina 

Rebased onto current master. Guys I'd be really grateful for merge as I must keep up my separate fork in order to maintain our rabbitmq cluster updated. Thank you

**Pull request merged**